packages/expo-linear-gradient/src/LinearGradient.tsx

```diff
@@ -33,7 +38,7 @@ export type LinearGradientProps = ViewProps & {
    * A readonly array of colors that represent stops in the gradient. At least two colors are required
    * (for a single-color background, use the `style.backgroundColor` prop on a `View` component).
    */
-  colors: readonly string[];
+  colors: LinearGradientColors;
```

Comment:
I believe this works too and maybe the docs output is a bit better, @vonovak?
```suggestion
  colors: readonly [string, string, ...string[]];
```

Reply:
@tsapeta the first problem seems to be readonly - even [the current docs](https://docs.expo.dev/versions/latest/sdk/linear-gradient/#colors) don't render it properly.

this: `colors: [string, string, ...string[]];` renders as below, so it's also a bit odd

<img width="596" alt="Screenshot 2024-08-08 at 9 43 59" src="https://github.com/user-attachments/assets/8a00d55b-efc4-4794-af45-a35b764dacb1">